internal/Picker/Picker.js

```diff
@@ -382,18 +493,47 @@ const PickerBase = kind({
 						aria-label={incrementAriaLabel}
 						className={css.secondaryItemIncrement}
 						disabled={isPenultimate}
-						onClick={handleSecondaryIncrement}
+						onClick={() => {
+							handleIncrement(); setTimeout(() => handleIncrement(), transitionDuration);
+						}}
 					>
-						<div className={css.label}>
-							{isPenultimate ? '' : secondaryIncrementValue}
-						</div>
+						<ViewManager
+							aria-hidden
+							arranger={arranger}
+							className={css.viewManager}
+							duration={transitionDuration}
+							index={Array.isArray(values) ? index + 2 : 0}
+							noAnimation={noAnimation || disabled}
+							reverseTransition={reverseTransition}
+						>
+							{secondaryIncrementValue()}
+						</ViewManager>
 					</PickerButtonItem>
 				}
 			</PickerRoot>
 		);
 	}
 });
 
+const ChangeAdapter = hoc((config, Wrapped) => {
```

Comment:
Could you please add comments for this new HoC? It looks better to add one for this even if it is private.